### PR TITLE
add fix and config commands to pre-commit-hooks

### PR DESCRIPTION
**Pull request by Cielquan**
closes #36

Discussion was in #36. New PR because of changing target branch.

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -5,3 +5,17 @@
   language: node
   types: [markdown]
   minimum_pre_commit_version: 0.15.0
+- id: markdownlint-cli2-config
+  name: markdownlint-cli2-config
+  description: "Checks the style of Markdown/CommonMark files with a non root config."
```
Comment (DavidAnson) [inline .pre-commit-hooks.yaml:10]: How about: "... with a non-root configuration file."

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -5,3 +5,17 @@
...
+  entry: markdownlint-cli2-config
+  language: node
+  types: [markdown]
+  minimum_pre_commit_version: 0.15.0
+- id: markdownlint-cli2-fix
+  name: markdownlint-cli2-fix
+  description: "Fix the style of Markdown/CommonMark files."
```
Comment (DavidAnson) [inline .pre-commit-hooks.yaml:17]: How about: "Checks and fixes"

**Review by DavidAnson — CHANGES_REQUESTED** [review]
Thanks for adding that! I left two nitpick comments.

File: .pre-commit-hooks.yaml (hunk 1 above)
Comment (Cielquan) [reply .pre-commit-hooks.yaml]: Sounds good. Will do.

File: .pre-commit-hooks.yaml (hunk 2 above)
Comment (Cielquan) [reply .pre-commit-hooks.yaml]: Sounds good. Will do.

**Review by DavidAnson — APPROVED** [review]

**Comment by DavidAnson** [comment]
Thanks!

**Pull request merged**